fgd/point/game/game_text.fgd

```diff
@@ -5,7 +5,7 @@
 	[
 	spawnflags(flags)  =
 		[
-		1: "All Players" : 0
+		1: "All Players" : 0 [+USE_MULTIPLAYER]
```

Comment:
Is this spawnflag (and the similar one in env_message) functional in momentum? iirc it is but I'm not sure about that.

Reply:
I don't know how I'd test it if there isn't ever true multiplayer in mmod

Reply:
I don't know how to test it because it seems the entities don't seem to work rn, I assume they're still on VGUI?

Reply:
The entity should work, but yeah the functionality shouldn't work in momentum as ghosts are not players, at least for the time being.